### ISSUE TITLE
Install gganimate failed

### ISSUE THREAD

**Issue by simonzhaoms** [gjwgit/animate#7]
When I install [`gganimate.tar.gz` in the repo](https://github.com/gjwgit/animate/blob/master/gganimate.tar.gz):
```console
$ Rscript -e "install.packages('gganimate.tar.gz', repos=NULL, lib='./R')"
Warning in untar2(tarfile, files, list, exdir, restore_times) :
  skipping pax global extended headers
Error in readRDS(pfile) : error reading from connection
* removing ‘/home/simon/.mlhub/animate/R/gganimate’
Warning message:
In install.packages("gganimate.tar.gz", repos = NULL, lib = "./R") :
  installation of package ‘gganimate.tar.gz’ had non-zero exit status
```

**Comment by gjwgit** [comment]
This should really be installed using 

`devtools::install_github('thomasp85/gganimate')`

Similarly ggflags:

 `devtools::install_github('ellisp/ggflags')`

**Comment by gjwgit** [comment]
Actual error in this case though I think is that tweenr has a new version 1.0.1 and animate failed installing the fixed version 1.0.0.

Need to install from the CRAN archive the current-1 version? Problem is with some of these packages not wanting to install with older version of R, probably for good reason though seem to be just fine. But we still want to do something?

**Comment by gjwgit** [comment]
Update tweenr version in the dependencies. Now working again. c94e5ebd3886cab6a0d2a5f3c520d39c91d1cb2d
